### PR TITLE
bug(notebook): fix docstore and query engine

### PR DESCRIPTION
**Pull request by vn01122003**
## Description

[Provide a brief description of the changes in this PR]

## How Has This Been Tested?

[Describe the tests you ran to verify your changes]

## Accepted Risk

[Any know risks or failure modes to point out to reviewers]

## Related Issue(s)

[If applicable, link to the issue(s) this PR addresses]

## Checklist:

- [ ] All of the automated tests pass
- [ ] All PR comments are addressed and marked resolved
- [ ] If there are migrations, they have been rebased to latest main
- [ ] If there are new dependencies, they are added to the requirements
- [ ] If there are new environment variables, they are added to all of the deployment methods
- [ ] If there are new APIs that don't require auth, they are added to PUBLIC_ENDPOINT_SPECS
- [ ] Docker images build and basic functionalities work
- [ ] Author has done a final read through of the PR right before merge

### CODE REVIEW

**Comment by lelouvincx** [comment]
Reviewing

File: chatbot-core/notebooks/POC1/POC1.ipynb
Comment (tuantran0910) [inline chatbot-core/notebooks/POC1/POC1.ipynb:1]:
```python

qdrant_client = QdrantClient(url="http://localhost:6333")
collection_name = "documents_collection"

collections = qdrant_client.get_collections()
print("Current collections:", collections)

if collection_name not in [collection.name for collection in collections.collections]:
```
Switch to Qdrant client method

**Review by tuantran0910 — COMMENTED** [review]
Hehe

**Comment by lelouvincx** [comment]
> Hehe

WTF

**Comment by lelouvincx** [comment]
Refer new notebook here: https://gist.github.com/lelouvincx/3bb1ffc8de21ef77ec024e04cea91db5

**Comment by lelouvincx** [comment]
This branch is staled and will not be merged into main. Please create another branch and PR to get latest chatbot-core code.

**Pull request closed without merging**